src/client/app/home/home-routing.module.ts

```diff
@@ -0,0 +1,13 @@
+import { NgModule } from '@angular/core';
+import { RouterModule } from '@angular/router';
+import { HomeComponent } from './home.component';
+
+@NgModule({
+  imports: [
+    RouterModule.forChild([
+      { path: 'about', component: HomeComponent }
```

Comment:
sorry about that. I wanted to see how angular deals with routing conflicts in Feature Modules (seems like first wins) and I accidentally did not switch it back. I'll create a fix